### PR TITLE
[Woo POS] M2: Update UI based on design for success screen

### LINKED ISSUE #11828: [Woo POS] M2: Update UI based on design for success screen
Follow up from this PR:

https://github.com/woocommerce/woocommerce-android/pull/11812#issuecomment-2195532317

Match design for success screen

### PR DESCRIPTION
**Pull request by backwardstruck**
Closes: #11828

### Description

Make UI closer the M2 i2 design per [this](https://wp.me/p91TBi-bHy#comment-12654) comment. 

### Steps to reproduce

1. Go to POS
1. Add a few products
1. Checkout
1. Complete the transaction

Confirm UI looks more like the new designs. Button should be white though. Icons may be changed in the future so no need to check those. 

### Images/gif

Should be close (OK if not 100% at this point) to the new design:

Screenshot 2024-07-29 at 17 26 26

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/woocommerce/woocommerce-android/milestone/228">19.8</a>. This milestone is due in less than 2 days.<br />
Please make sure to get it merged by then or assign it to a milestone with a later deadline.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by backwardstruck** [comment]
FYI @samiuelson I updated this based on the meeting today.

**Review by samiuelson — COMMENTED** [review]
Looks good overall. I can see weird paddings at the sides of the screen though:
Screenshot_20240801_172906
I think it would be nice to remove these.

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/home/WooPosHomeScreen.kt [hunk 1]
```diff
@@ -128,17 +128,6 @@ private fun WooPosHomeScreen(
         label = "cartOverlayAnimated"
     )
 
-    val totalsStartPaddingAnimatedDp by animateDpAsState(
```
Comment (backwardstruck) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/home/WooPosHomeScreen.kt:131]: This was added [here](https://github.com/woocommerce/woocommerce-android/pull/11879) but as far as I can tell, we don't need it anymore. If that's not the case I will put it back. FYI @kidinov

**Comment by backwardstruck** [comment]
> Looks good overall. I can see weird paddings at the sides of the screen though: Screenshot_20240801_172906 I think it would be nice to remove these.

Fixed in [this](https://github.com/woocommerce/woocommerce-android/pull/12165/commits/c4789e2defb0731f4f61e17ed27255e678e6b78b) commit.

**Review by samiuelson — APPROVED** [review]
Looks great! 🚀

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/home/totals/payment/success/WooPosTotalsPaymentSuccessScreen.kt [hunk 2]
```diff
@@ -43,33 +38,17 @@ fun WooPosPaymentSuccessScreen(
     onNewTransactionClicked: () -> Unit,
 ) {
     Column(
-        modifier = Modifier.fillMaxSize(),
+        modifier = Modifier
+            .fillMaxSize()
+            .background(Color(0xFF98F179)),
```
Comment (kidinov) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/home/totals/payment/success/WooPosTotalsPaymentSuccessScreen.kt:43]: @backwardstruck I think we should not use hardcoded colors at this stage as it won't work in the dark mode

Comment (backwardstruck) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/ui/woopos/home/totals/payment/success/WooPosTotalsPaymentSuccessScreen.kt:43]: Agreed. I'll start adding them to the xml file instead.

**Pull request merged**